src/Symfony/Component/Process/Process.php

```diff
@@ -422,6 +429,8 @@ public function signal($signal)
     public function disableOutput()
     {
         if ($this->isRunning()) {
+            $this->stop(0);
```

Comment:
reverted, even if I think that stopping the process when an exception is triggered would make things more robust (less process left behind)